distributed/protocol/skorch.py

```diff
@@ -0,0 +1,28 @@
+import cloudpickle
+import skorch
+from .serialize import dask_serialize, dask_deserialize
+
+@dask_serialize.register(skorch.NeuralNet)
+def serialize_skorch(x):
+    has_module = hasattr(x, "module_")
+    headers = {"has_module": has_module}
+    if has_module:
+        module = x.__dict__.pop("module_")
+        try:
```

Comment:
What happens if there is an exception here? What exceptions might occur?

Reply:
I have not run into any so removed this clause. Will add back if i run into it again. I just used the same code as a first pass.